TelepresenceBot/movement-unit/movement-unit.ino

```diff
@@ -0,0 +1,140 @@
+#include <AFMotor.h> // from https://github.com/adafruit/Adafruit-Motor-Shield-library
+#include <Servo.h> 
+
+AF_DCMotor motorRight1(3);
+AF_DCMotor motorRight2(2);
+AF_DCMotor motorLeft1(4);
+AF_DCMotor motorLeft2(1);
+
+const char COMMAND_FORWARD = 'w';
+const char COMMAND_BACKWARD = 's';
+const char COMMAND_TEST = 't';
+const int MAX_SPEED = 255;
+const int DELTA_SPEED = 50;
+
+int currentDirection = RELEASE;
+int currentSpeed = 0;
+
+void setup() {
+  Serial.begin(9600);           
+  Serial.println("Movement unit starting");
+}
+
+void loop() {
+  if (Serial.available() == 0) {
+    return;
+  }
+  
+  char inChar = Serial.read();
+
+  switch(inChar) {
+    case (COMMAND_FORWARD):
+      increaseForward();
+      break;
+    case (COMMAND_BACKWARD):
+      increaseBackward();
+      break;
+    case (COMMAND_TEST):
+      testMotors();
+      break;
+    default:
+       Serial.println("Unknown command " + inChar);
+  }
+}
+
+void testMotors() {
+  int i;
+  
+  setRightDirection(FORWARD);
+  setLeftDirection(FORWARD);
+  for (i=0; i<255; i++) {
+    setRightSpeed(i);  
+    setLeftSpeed(i);  
+    delay(3);
+  }
+
+    setRightSpeed(255);  
+    setLeftSpeed(255);   
+    delay(5000);
+ 
+ 
+  for (i=255; i!=0; i--) {
+    setRightSpeed(i);  
+    setLeftSpeed(i);  
+    delay(3);
+  }
+ 
+  setRightDirection(BACKWARD);
+  setLeftDirection(BACKWARD);
+  for (i=0; i<255; i++) {
```

Comment:
Yeah, I wouldn't worry too much about what's in `testMotors()`.
We can remove it once we fix all the power problems and we have a final schematics